sample-keys/create-ca.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 mkdir -p CA
 # creat key if none exists
+echo "create key"
 [[ -z CA/rootCA.key ]] && openssl genrsa -out CA/rootCA.key 4096
```

Comment:
This will always return false: https://www.shellcheck.net/wiki/SC2157
```suggestion
nameCA="CA/rootCA"
keyCA="$nameCA.key"
[[ -z CA/rootCA.key ]] && openssl genrsa -out CA/rootCA.key 4096
echo "create key $keyCA"
[[ ! -f "$keyCA" ]] && openssl genrsa -out "$keyCA" 4096
```